pkg/ibmvpc/parser.go

```diff
@@ -447,7 +447,7 @@ func getInstancesConfig(
 			res[vpcUID].Nodes = append(res[vpcUID].Nodes, intfNode)
 			res[vpcUID].NameToResource[intfNode.Name()] = intfNode
 			vsiNode.nodes = append(vsiNode.nodes, intfNode)
-			intfNameToIntf[*netintf.Name] = intfNode
+			intfNameToIntf[intfNode.ResourceName] = intfNode
```

Comment:
This change seems to have no effect; the two properties are identical. The point is just referring to the original property? 
Its not specifically related to #237, right?